docs/src/main/sphinx/installation/rpm.rst

```diff
@@ -11,9 +11,10 @@ to start as a service.
 
 ..  warning::
 
-  It is recommended to deploy Trino with the Helm chart on Kubernetes or
-  manually with the :doc:`Docker containers <containers>` or the :doc:`tar
-  archive <deployment>`. While the RPM is available for use, it is not tested.
+  It is recommended to deploy Trino with the :doc:`Helm chart <kubernetes>` on
+  Kubernetes or manually with the :doc:`Docker containers <containers>` or the
+  :doc:`tar archive <deployment>`. While the RPM is available for use, it is not
+  tested.
```

Comment:
> is not tested

well, https://github.com/trinodb/trino/blob/master/core/trino-server-rpm/src/test/java/io/trino/server/rpm/ServerIT.java does exist

i think the docker container or helm charts may not be tested more than that, are they?

we should change to "not supported" or "discouraged".